gcc/rust/typecheck/rust-hir-type-check-expr.cc

```diff
@@ -1152,7 +1152,7 @@ TypeCheckExpr::visit (HIR::MethodCallExpr &expr)
       rust_debug_loc (expr.get_method_name ().get_generic_args ().get_locus (),
 		      "applying generic arguments to method_call: {%s}",
 		      lookup->debug_str ().c_str ());
-      HIR::GenericArgs &args = expr.get_method_name ().get_generic_args ();
+      HIR::GenericArgs args = expr.get_method_name ().get_generic_args ();
```

Comment:
Why take a copy here?

Reply:
This was due to an invalid warning upstream which I believe has now been fixed. I'll have a look.